monitor/main.go

```diff
@@ -72,6 +72,27 @@ func main() {
 	execute()
 }
 
+// buildServerOrExit opens a listener socket at path. It exits with logging on
+// all errors.
+func buildServerOrExit(path string) net.Listener {
+	scopedLog := log.WithField(logfields.Path, path)
+
+	os.Remove(path)
+	server, err := net.Listen("unix", path)
+	if err != nil {
+		scopedLog.WithError(err).Fatal("Cannot listen on socket")
```

Comment:
I think the message reflects the function being called. In any case, it is correct since so far, all it is doing is listening. The writing part comes later, when something connects.